CONTRIBUTING.md

```diff
@@ -87,23 +61,67 @@ A: There is no strict definition however examples of substantive changes are:
+A: Bring the idea to the [#bfd](https://cmsgov.slack.com/archives/C010WDXAZFZ) channel or the Scrum of Scrums meeting and talk it over with the core team.
 
 
 Q: What if my RFC is not accepted?
 
-A: It will be closed and can be reopened if it is updated to address the items that prevented it's acceptance.
+A: It will be closed, but can be reopened if it is updated to address the items that prevented it's acceptance.
 
 
 Q: What if my team doesn’t have the resources to implement our accepted RFC? 
 
 A: Anyone can coordinate with you and the core team to take over the work. 
 
+### Getting started
+
+Clone this repository to get started.
+
+```
+git clone https://github.com/CMSgov/beneficiary-fhir-data.git
+```
+
+Then checkout a new branch to work on (the `-b` tells it to make a new branch).
+
+```
+git checkout -b <your username>/<your feature name>
+# If you ever have to rename this branch you can do so with
+# git branch -m <your username>/<your feature name> <your username>/<your new name>
+```
+### Setting Up A Local Development Environment On Windows
+
+Going to work on this project or one of the other Blue Button Java projects? Great! You can follow the instructions in [Development Environment Setup](./dev/devenv-readme.md) to get going.
+
+#### Opening A PR
+
+To contribute work back to the BFD your branch must be pushed to the `CMSgov/beneficiary-fhir-data` repository.
+```
+git push origin <your username>/<your feature name>
+# To make sure "origin" points to CMSgov/beneficiary-fhir-data run
+# git remote -v
+# if a different remote points to CMSgov/beneficiary-fhir-data
+# replace "origin" with that remotes name in the origional command
+```
+In order to obtain permission to do this contact a github administrator or reach out on [#bfd](https://cmsgov.slack.com/archives/C010WDXAZFZ).
+Once pushed, [open a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) and use this [PR template](https://github.com/CMSgov/cms-oeda-dasg) found in .github
+
+Please fill out each section of the body of the PR or quickly justify why the section does not apply to this particular change.
+Reviewers will automatically be suggested or requested based on the projects CODEOWNERS file, feel free to add other reviewers if desired.
+Once all automated checks pass and two reviewers have approved your pull request, a code owner from the core team will do a final review and make the decision to merge it in or not.
+
+If you have any questions feel free to reach out on the [#bfd](https://cmsgov.slack.com/archives/C010WDXAZFZ) channel in CMS slack!
+
+**FAQ**
+
+Q: What if the core team rejects my PR?
+
+A: The BFD core team will commit to never rejecting a PR without providing a path forward. The developer who put up the PR should review any feedback, and discuss with their product manager the scope of the work that is now outstanding.
+
 Setting up the FHIR server locally
```

Comment:
I think we wanted to move this from the contributing and into the readme. Lines 125 through 288 need a new home, I believe.

Reply:
I'm happy to move those to the README instead - @cbrunefearless @keithdadkins does that work?